0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -590,10 +590,8 @@ Metadata describing the ledger. This data is returned by the [`getInfo`](#getinf
 | Type | Name | Description |
 |:--|:--|:--|
 | `String` | [prefix](#prefix) | The plugin's ILP address prefix |
-| `Number` | [precision](#precision) | Total number of digits allowed |
-| `Number` | [scale](#scale) | Digits allowed after decimal |
-| `String` | [currencyCode](#currencycode) | ISO three-letter currency code |
-| `String` | [currencySymbol](#currencysymbol) | UTF8 currency symbol |
+| `String` | [currencyCode](#currencyCode) | ISO three-letter currency code |
+| `Number` | [currencyScale](#currencyScale) | Integer `(..., -2, -1, 0, 1, 2, ...)`, such that `1 <currencyCode>` equals `10^<currencyScale>` of the ledger's base units
```

Comment:
Maybe swap this around so `1 base unit equals 10^(-1 * <currencyScale>) <currencyCode>`. In the current phrasing, a negative scale would imply that a fractional amount of base units would be used to represent one <currencyCode>, which isn't possible.